src/frontend/src/app/components/event/event.component.html

```diff
@@ -0,0 +1,17 @@
+ <mat-card class="eventCard">
+    <mat-card-header>
+      <div mat-card-avatar class="example-header-image"></div>
```

Comment:
What are your plans for the avatar piece? Looking at this, it looks like it might make more sense to just have `<div class="example-header-image"></div>` here.

Reply:
Hopefully we can define a set of images for each event type and/or organization and then apply then bind them based on that event type. 

This is just the material demo I found to get things started so we can see what is possible.